allennlp/service/index.html

```diff
@@ -4,56 +4,96 @@
 </head>
 <body>
-    <form>
-        <div>
-            <select name="model-picker" id="model-picker">
-            </select>
-        </div>
-        <div>
-            <textarea name="model-input" id="model-input"></textarea>
-        </div>
-        <div>
-            <button type="button" onclick="runModel()">run model</button>
-        </div>
-    </form>
-    <div id="output">
-    </div>
+    <div id="bidaf">
+        <h2>bidaf</h2>
+        <form id="bidaf-form">
+            <h3>Passage</h3>
+            <div>
+                <textarea name="bidaf-passage" id="bidaf-passage">The Seahawks are the only football team that plays in Seattle.</textarea>
+            </div>
+            <h3>Question</h3>
+            <div>
+                <textarea name="bidaf-question" id="bidaf-question">Which football team plays in Seattle?</textarea>
+            </div>
+            <div>
+                <button type="button" onclick="runBidaf()">run bidaf</button>
+            </div>
+        </form>
+        <div id="bidaf-output"></div>
+    </div>  <!-- bidaf -->
 
-    <script>
+    <hr>
 
-/* Populate the dropdown based on available models. */
-(function() {
-    var xhr = new XMLHttpRequest();
-    xhr.open('GET', 'models');
-    xhr.onload = function() {
-        if (xhr.status == 200) {
-            var models = JSON.parse(xhr.responseText)['models'];
-            models.forEach(function(modelName) {
-                var option = document.createElement("option");
-                option.value = modelName;
-                option.innerText = modelName;
-                document.getElementById("model-picker").appendChild(option);
-            });
-        }
-    }
-    xhr.send();
-})();
+    <div id="snli">
+        <h2>decomposable attention</h2>
+        <form id="snli-form">
+            <h3>Premise</h3>
+            <div>
+                <textarea name="snli-premise" id="snli-premise">AllenNLP is the premier library for doing state of the art natural language processing.</textarea>
+            </div>
+            <h3>Hypothesis</h3>
+            <div>
+                <textarea name="snli-hypothesis" id="snli-hypothesis">AllenNLP is good at natural language processing</textarea>
+            </div>
+            <div>
+                <button type="button" onclick="runSnli()">run decomposable attention</button>
+            </div>
+        </form>
+        <div id="snli-output"></div>
+    </div> <!-- snli -->
 
-/* Run the selected model and put the results in the `output` div. */
-function runModel() {
-    var modelName = document.getElementById("model-picker").value;
-    var input = document.getElementById("model-input").value;
+    <hr>
 
+    <div id="srl">
+        <h2>semantic role labeling</h2>
+        <form id="srl-form">
+            <h3>Sentence</h3>
+            <div>
+                <textarea name="srl-sentence" id="srl-sentence">Beavers build dams in order to impress technical recruiters.</textarea>
```

Comment:
I think the current SRL prediction interface requires specifying the verb, also.